src/js/utils/events.js

```diff
@@ -84,6 +84,9 @@ function _handleMultipleEvents(fn, elem, types, callback) {
  *         Fixed event object.
  */
 export function fixEvent(event) {
+  if (event.fixed_) {
```

Comment:
This costs us 0.1 ms per event, but some event have it done to them 5+ times. Might as well only do it once.

Reply:
Do we always go into the line 101/104 block?

Reply:
It seems like we almost always do yeah.